packages/opds-web-client/src/components/Collection.tsx

```diff
@@ -47,7 +47,7 @@ export default class Collection extends React.Component<CollectionProps, {}> {
     return (
       <div className="collection">
         { hasFacets && (
-          <div className="facet-groups" role="complementary" aria-label="filters">
+          <div className="facet-groups" aria-label="filters">
```

Comment:
If "complementary" isn't right, maybe this should have some other role to make it easier to navigate to.

Reply:
This makes sense as an `aside` (instead of a div) and I'll try that

Reply:
Aside doesn't work either. This is the issue: https://dequeuniversity.com/rules/axe/3.3/landmark-complementary-is-top-level

Removing the `role="complementary"` passes the aXe test. I think this should be its own ticket which can be worked on later since this is a bigger change. It'll require updating the app so that this section is no longer in the main container and be able to be an aside or have the "complementary" role. What do you think about this, rather than updating it in this PR?